crates/project_model/src/project_json.rs

```diff
@@ -91,11 +91,8 @@ impl ProjectJson {
     pub fn crates(&self) -> impl Iterator<Item = (CrateId, &Crate)> + '_ {
         self.crates.iter().enumerate().map(|(idx, krate)| (CrateId(idx as u32), krate))
     }
-    pub fn path(&self) -> Option<&AbsPath> {
-        match &self.project_root {
-            Some(p) => Some(p.as_path()),
-            None => None,
-        }
+    pub fn path(&self) -> &AbsPath {
```

Comment:
(short doc comment on this method would be nice too)

Reply:
Done.